modules/updating-eus-to-eus-upgrade-cli.adoc

```diff
@@ -125,7 +125,7 @@ $ oc patch mcp/worker --type merge --patch '{"spec":{"paused":false}}'
 +
 [IMPORTANT]
 ====
-If pools are not unpaused, the cluster is not permitted to update to any future minor versions, and maintenance tasks such as certificate rotation are inhibited. This puts the cluster at risk for future degradation.
+If pools are not unpaused, the cluster is not permitted to update to any future minor versions, and maintenance tasks are inhibited. This puts the cluster at risk for future degradation.
```

Comment:
The same "IMPORTANT" note is present in `modules/updating-eus-to-eus-upgrade-console.adoc`. But it says:

```suggestion
If pools are not unpaused, the cluster is not permitted to update to any future minor versions, and some maintenance tasks are inhibited. This puts the cluster at risk for future degradation.
```